apps/system/js/notification_screen.js

```diff
@@ -110,20 +106,20 @@ var NotificationScreen = {
 
     var self = this;
     SettingsListener.observe('notification.ringtone', '', function(value) {
-      self._sound = self.ringtoneURL.set(value);
+      LazyLoader.load(['shared/js/settings_url.js']).then(function() {
+        self.ringtoneURL = new SettingsURL();
```

Comment:
Do we want to always create a new `SettingsURL` here? To keep parity with how it worked before, I think we want to only create the `ringtonURL` once, and call set on it when receiving this event.